Source/WebInspectorUI/UserInterface/Views/Popover.js

```diff
@@ -49,6 +49,14 @@ WI.Popover = class Popover extends WI.Object
         this._container.className = "container";
 
         this._drawBackgroundAnimationIdentifier = undefined;
+
+        // Background canvas
+        this._canvas = document.createElement("canvas");
+        this._canvas.style.position = "absolute";
+        this._canvas.style.top = 0;
+        this._canvas.style.left = 0;
+        this._canvas.style.zIndex = -1;
```

Comment:
We should make sure style continues to be defined in CSS, so instead the `_canvas` element should get a class like `background-canvas` and have these style bits come from CSS for `.popover.background-canvas`.